src/Microsoft.PowerPlatform.PowerApps.Persistence/Yaml/SerializerBuilderExtensions.cs

```diff
@@ -10,6 +10,8 @@ namespace Microsoft.PowerPlatform.PowerApps.Persistence.Yaml;
 
 internal static class SerializerBuilderExtensions
 {
+    private const string ControlPropertyName = "Control";
```

Comment:
Probably builder extension is not the best place for this constant

Reply:
agreed. But we're not referencing it anywhere else, and we use "Control" only when serializing into YAML for now. I'd move it outside when we need to